src/Core/Cache/ManifestCacheFactory.php

```diff
@@ -45,22 +48,28 @@ public function create($service, array $params = [])
     {
         // Override default cache generation with SS_MANIFESTCACHE
         $cacheClass = Environment::getEnv('SS_MANIFESTCACHE');
+
         if (!$cacheClass) {
+            $params['useInjector'] = false;
             return parent::create($service, $params);
         }
 
+        $cacheClass = $this->convertLegacySymfonyClassName($cacheClass);
+
         // Check if SS_MANIFESTCACHE is a factory
         if (is_a($cacheClass, CacheFactory::class, true)) {
             /** @var CacheFactory $factory */
             $factory = new $cacheClass;
             return $factory->create($service, $params);
```

Comment:
Don't we need to pass in `$params['useInjector'] = false;` here as well?

Reply:
Nope, they always use Injector - e.g. https://github.com/silverstripe/silverstripe-framework/blob/4/src/Core/Cache/MemcachedCacheFactory.php#L34

Reply:
I don't see what the `MemcachedCacheFactory` has to do with it? This is for `ManifestCacheFactory` and according to a comment you've added to `DefaultCacheFactory` the only reason we don't use injector for the other instantiations done by this factory is "because config is not available at that point"... if the config isn't available above or below where we're passing `useInjector` as false, then why is it available here?

Reply:
:facepalm: Sorry, think I confused myself

Yes, it should have `useInjector` as false, for the scenario where someones uses DefaultCacheFactory as the value in `SS_MANIFESTCACHE`

I've simply moved the `$params['useInjector'] = false;` so that $params is set like that for whatever object in instantiated